AutoSteamApp/Helpers/StaticHelpers.cs

```diff
@@ -0,0 +1,179 @@
+﻿using AutoSteamApp.Configuration;
+using AutoSteamApp.ProcessMemory;
+using GregsStack.InputSimulatorStandard;
+using GregsStack.InputSimulatorStandard.Native;
+using Logging;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+
+namespace AutoSteamApp.Helpers
+{
+    public static class StaticHelpers
+    {
+        /// <summary>
+        /// Returns the process which contains the process name as defined in the fields
+        /// </summary>
+        /// <returns></returns>
+        public static Process GetMHWProcess()
+        {
+            //Retrieve all processes with defined process name
+            var processes = Process.GetProcessesByName(MHWMemoryValues.ProcessName);
+            // Try to return the first one
```

Comment:
You can always make the code look cleaner if you introduce empty lines between statements (A comment should also be preceded by an empty line). Another good strategy to improve readability is to split multi-statement `if` clauses on separate lines, based on DeMorgan's laws - this way you can also understand the statement easier.